src/uu/stty/src/stty.rs

```diff
@@ -274,6 +307,15 @@ fn print_terminal_size(termios: &Termios, opts: &Options) -> nix::Result<()> {
     Ok(())
 }
 
+fn is_control_char(option: &str) -> Option<SpecialCharacterIndices> {
```

Comment:
This function name is a bit misleading: functions starting with `is_` usually return a `bool`.